queries/c_sharp/highlights.scm

```diff
@@ -366,9 +439,11 @@
   ] @punctuation.bracket)
 
 [
-  (this_expression)
-  (base_expression)
+  "assembly"
+  "module"
   "this"
+  "base"
```

Comment:
Right now this gets `@constructor` and `@variable.builtin`, not sure if this is intended

Reply:
```cs
public DerivedClass() : base()
    {
    }
```

vs `base.foo` (similar to `this.foo`/`self.foo` in other languages)

but you bring up a good point, this should be lower in the list to override the catch-all base query